elasticdl_client/common/k8s_client.py

```diff
@@ -29,6 +33,21 @@
 ELASTICDL_REPLICA_INDEX_KEY = "elasticdl-replica-index"
 
 
+class PodType(Enum):
```

Comment:
Maybe we can declare the enumerated PodType string just like the following way in constants.py

```python
class DistributionStrategy(object):
    LOCAL = "Local"
    PARAMETER_SERVER = "ParameterServerStrategy"
    ALLREDUCE = "AllreduceStrategy"
```

Reply:
Done